src/chapters/loops/for-loops.rst

```diff
@@ -0,0 +1,90 @@
+**for** Loops
+=============
+
+.. index::
+   single: loop; for
+   single: iteration; definite
+
+.. index:: ! for loop
+
+The ``for`` loop is the first JavaScript tool for iteration that we will explore. A **for loop** allows us to carry out **definite iteration**. Definite iteration is the process of repeating a specific task with a specific data set. In other words, when a ``for`` loop begins it can usually be determined exactly how many times it will execute: once for each item in the data set.
+
+**for** Loop Syntax
+-------------------
+
+.. index::
+   single: for loop; syntax
+
+We have already seen the basic syntax of a ``for`` loop.
+
+.. sourcecode:: js
+
+   for (let i = 0; i < 51; i++) {
+      console.log(i);
+   }
+
+This program prints the integers 0 through 50. In the language of definite iteration, we can say that the operation that this loop carries out printing a value to the console, and its data set is the collection of integers 0...50.
```

Comment:
Grammar.  How about, “This program prints the integers 0 through 50, one number per line. In the language of definite iteration, we say that the loop has a data set of 0-50, and its action is to print a value to the console.”